perf/Cargo.toml

```diff
@@ -13,6 +13,7 @@ edition = { workspace = true }
 ahash = { workspace = true }
 bincode = { workspace = true }
 bv = { workspace = true, features = ["serde"] }
+crossbeam-channel = { workspace = true }
```

Comment:
kind of arbitrary extra dep addition, but i think it's okay for better code organization. alternatively, we can define `BankingPacketBatch`/`BankingPacketReceiver` at `solana-unified-scheduler-pool` without a single dep graph chagne, but it felt me that it's off-place.

Reply:
Not a big fan of putting it here. `perf` seems isolated from any communication, so doesn't seem correct to me to add this just for a type-alias.

Maybe now is a good time to just set up a more (imo) reasonable structure of crates? (doesn't matter THAT much since both banking and sigverify are in `core`)
A typical pattern that I've seen for crates is something like this:

```tree
/
├── stage1/
├── communication/
└── stage2/
```

The communication crate is separate from both the upstream and downstream consumer(s), and they both just need to follow the interface defined in `communication`. Think it is a step toward separating sigverify & banking code from core as well.

Reply:
These would be specific to agave client, so `agave-` prefix is more appropriate for naming imo.
If it's just going to be communication between SigVerifyStage (possibly soon to be renamed VerifyStage) and BankingStage (should we rename this to BlockProductionStage?) I wonder if the name `core-types` is too generic - there's a LOT of other stuff in `core`

Reply:
hmm, how about `agave-banking-stage-types`, then? i thought we won't rename replaying-stage to block-verification-stage, so picked the `banking-stage` likewise

Reply:
I think it's still better to have some sort of `comm` or something in the name.
`banking-stage-types`, at least to me, implies they are types internally used in banking stage; which isn't really the case here. They are simply the types used for banking stage ingress.

`agave-banking-stage-ingress-types` too verbose? terrible naming?

Side-note, I have considered to rename banking-stage to block-production if we break it out into its' own crate.